src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -1007,6 +1007,9 @@ int FindUnitNumber(std::string const &FileName) // File name to be searched.
         std::string::size_type TestFileLength;
         std::string::size_type Pos; // Position pointer
         for (UnitNumber = 1; UnitNumber <= MaxUnitNumber; ++UnitNumber) {
+            // Skip preassigned units - gio::inquire breaks std::cout on Windows - see GlobalStreams::
+            if ((UnitNumber == 0) || (UnitNumber == 5) || (UnitNumber == 6) || (UnitNumber == 100) || (UnitNumber == 101) || (UnitNumber == 102))
+                continue;
```

Comment:
Could we enclose this line in braces so it is clear that the `continue` goes with the `if` and not the block below?

Reply:
OK - added braces and updated comment which was incomplete before. Ready again.